### DISCUSSION TITLE
sats amount with whitespace separator `XXX XXX XXX`

### DISCUSSION THREAD

**Discussion by ghost** [btcpayserver/btcpayserver#2967] [General]
Currently, the sats denomination is displayed without separators `XXXXXXXXX`. It is confusing to read such a long string of unbroken text.

Instead, add a separator to the denomination `XXX XXX XXX`. This is easier to read. Whitespace also removes the uncertainty of `.` / `,` separators.

---

Notice this can also be done for the btc denomination, `XX XXX XXX.XXXX XXXX`

**Comment by bolatovumar** [comment]
@MaxHillebrand seems like it's displayed with spaces in the invoice. Could you clarify where it's displayed without any separators?

![Capture](https://user-images.githubusercontent.com/1934678/138579630-c3540c1f-88cf-49e6-8c5d-17af54bd0447.PNG)

**Reply by ghost** [reply]
Good catch, I believe it's not in the crowdfund app.

**Reply by bolatovumar** [reply]
@MaxHillebrand ok, got it. PR up here: https://github.com/btcpayserver/btcpayserver/pull/3011.
